### DISCUSSION TITLE
Feature Request: Sidebar Display for Trace Details in Dataset Runs

### DISCUSSION THREAD

**Discussion by tuan3w** [langfuse/langfuse#2523] [Ideas]
### Describe the feature or potential improvement

When checking a dataset run, I want to see details about failure cases. However, opening traces in another tab makes it hard for me to switch between different traces and the overview link. I would love to have a feature that opens trace details in the sidebar to check the details.
![image](https://github.com/langfuse/langfuse/assets/2762678/80542bbd-3f0d-49d3-b748-0ca8ba3dbb50)

I think this would be a very helpful feature for debugging problems for many people.
Thanks.

**Comment by marcklingen** [comment]
thanks for raising this. Do you like the sidebar that's opened when viewing span inputs/outputs on the trace timeline?

**Comment by justingodden** [comment]
Hi, if I understand the OP correctly, just to say I also think the ability to inspect/expand traces in the traces list page, rather than needing to open the specific trace into it's own Trace Detail page would be useful.

Either something like an arrow expander that if you click, expands the trace tree underneath to show child spans' observable name, input output etc. Kind of like how LangSmith does it. Or could re-use the sidebar in the Trace Detail view and have that pop in

**Comment by tuan3w** [comment]
Hi all.
I prefer to open the sidebar to view the trace details. A split view is also a good option, I think.
